src/shared/App.tsx

```diff
@@ -20,7 +21,8 @@ const App: React.FC = () => {
   // use it as the location for the <Switch> so
   // we show the gallery in the background, behind
   // the modal.
-  const background = location.state && location.state.background;
+  const background =
```

Comment:
Are there any changes in Location/history packages? 
location.state.background vs location.state as { background?: Location }).background (?)

Reply:
The history @types package was changed, yeah :)